site/templates/items/meta_item.jet

```diff
@@ -61,11 +69,17 @@
             {{ end }}
 
             <div class="buttons">
-              <s72-play-button data-slug="{{item.Slug}}" title="{{item.Title}}"></s72-play-button>
-              <s72-pricing-buttons data-slug="{{item.Slug}}" class="pricing-buttons-fit" title="{{item.Title}}"></s72-pricing-buttons>
-              {{ if item.ItemType == "film" }}
+              {{if showPlayButton}}
+                <s72-play-button data-slug="{{item.Slug}}" title="{{item.Title}}"></s72-play-button>
+              {{end}}
+
+              {{if showPricingButtons}}
+                <s72-pricing-buttons data-slug="{{item.Slug}}" class="pricing-buttons-fit" title="{{item.Title}}"></s72-pricing-buttons>
+              {{end}}
+
+              {{if isFilm}}
```

Comment:
Will this work for tv episodes (also watchable)

Reply:
Yeah we only show the can-be-watched-button for films.